kernel_config

```diff
@@ -3666,8 +2746,7 @@ CONFIG_FANOTIFY=y
 # CONFIG_QUOTA is not set
 # CONFIG_QUOTACTL is not set
 # CONFIG_AUTOFS4_FS is not set
-CONFIG_FUSE_FS=y
-# CONFIG_CUSE is not set
+# CONFIG_FUSE_FS is not set
```

Comment:
Is there any benefit to b2d having modules over built in? either way, we're using iso-space?